pkgs/servers/web-apps/lemmy/ui.nix

```diff
@@ -62,6 +62,7 @@ mkYarnPackage {
   buildPhase = ''
     # Yarn writes cache directories etc to $HOME.
     export HOME=$PWD/yarn_home
+    export COMMIT_HASH=${src.rev}
```

Comment:
This doesn't work, as the COMMIT_HASH variable gets overridden in the package.json.

You'd have to either patch the package.json, or go the "fake git binary" route that I mentioned in the original PR.

Reply:
@n3oney let me know if the new commit solves your issue then.

Reply:
```
error: builder for '/nix/store/f6nf4ljp1il7ah4ilfwbzla00mi4hylx-lemmy-ui.drv' failed with exit code 1;
       last 8 log lines:
       > unpacking sources
       > unpacking source archive /nix/store/ixhd4ks8fw7jprbmr34g88r0phn79009-source
       > source root is source
       > patching sources
       > updateAutotoolsGnuConfigScriptsPhase
       > configuring
       > building
       > substitute(): ERROR: file 'package.json' does not exist
       For full logs, run 'nix log /nix/store/f6nf4ljp1il7ah4ilfwbzla00mi4hylx-lemmy-ui.drv'.
       ```